crates/nu-cli/src/commands/math/command.rs

```diff
@@ -41,7 +37,7 @@ mod tests {
     use crate::commands::math::{
         average::average, max::maximum, min::minimum, utils::MathFunction,
     };
-    use nu_plugin::test_helpers::value::{decimal, int};
+    use nu_plugin::test_helpers::value::{decimal, int, table};
```

Comment:
Looks like this is causing the CI failure, since `table` isn't getting used.

Reply:
Yeah, it's a bit frustrating my editor doesn't help me with this.
Definitely miss some of the tooling that came with Go in VSCode